cookies/tests/test_operations.py

```diff
@@ -180,3 +180,63 @@ def test_merge_updates_relations(self):
 
     def tearDown(self):
         ConceptEntity.objects.all().delete()
+
+
+class TestAddResourcesToCollection(unittest.TestCase):
+    def test_add_one(self):
+        """
+        Only one :class:`.Resource` instance should be added to a collection.
+        """
+        resource = Resource.objects.create(name='first_resource')
+        qs_resource = Resource.objects.all()
+        collection_before = Collection.objects.create(name='first_collection')
+        collection_after = operations.add_resources_to_collection(qs_resource, collection_before)
+
+        assert resource in collection_after.resources.all(), "resource not added to collection\
```

Comment:
Be sure to use the UnitTest class methods. If this assertion fails, it will raise an AssertionError and the test will exit with an Error; whereas we want the test to Fail. In this case, use ``assertIn``:

```python
self.assertIn(resource, collection_after.resources.all(), "resource not added to collection add_resources_to_collection operation not performed")
```